packages/react-geo/jest.config.js

```diff
@@ -2,10 +2,10 @@ module.exports = {
   collectCoverage: true,
   collectCoverageFrom: ['<rootDir>/src/**/*.(ts|tsx)'],
   // ignore coverage for top level "export", version and style files
```

Comment:
Can we remove version here as well?
```suggestion
  // ignore coverage for top level "export" and style files
```

Reply:
Did not notice that. Removed.